hummingbot/connector/exchange/bloxroute_openbook/bloxroute_openbook_web_utils.py

```diff
@@ -0,0 +1,63 @@
+import time
+from typing import Callable, Optional
+from urllib.parse import urljoin
+
+import bxsolana
+
+import hummingbot.connector.exchange.bloxroute_openbook.bloxroute_openbook_constants as CONSTANTS
+from hummingbot.connector.time_synchronizer import TimeSynchronizer
+from hummingbot.connector.utils import GZipCompressionWSPostProcessor, TimeSynchronizerRESTPreProcessor
+from hummingbot.core.api_throttler.async_throttler import AsyncThrottler
+from hummingbot.core.web_assistant.auth import AuthBase
+from hummingbot.core.web_assistant.connections.data_types import RESTMethod
+from hummingbot.core.web_assistant.web_assistants_factory import WebAssistantsFactory
+
+
+def public_rest_url(path_url: str, domain: str = CONSTANTS.DEFAULT_DOMAIN) -> str:
```

Comment:
seems like a pretty useful file. Can you get away with not using it?

Reply:
Yeah, I am going to delete most of it, but there is one function called `get_current_server_time` that is used by the base exchange class